keras_nlp/models/whisper/whisper_backbone.py

```diff
@@ -0,0 +1,293 @@
+# Copyright 2023 The KerasNLP Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Whisper backbone model."""
+
+
+import tensorflow as tf
+from tensorflow import keras
+
+from keras_nlp.layers.position_embedding import PositionEmbedding
+from keras_nlp.layers.token_and_position_embedding import (
+    TokenAndPositionEmbedding,
+)
+from keras_nlp.models.backbone import Backbone
+from keras_nlp.models.whisper.whisper_decoder import WhisperDecoder
+from keras_nlp.models.whisper.whisper_encoder import WhisperEncoder
+
+
+def whisper_kernel_initializer(stddev=0.02):
+    return keras.initializers.TruncatedNormal(stddev=stddev)
+
+
+@keras.utils.register_keras_serializable(package="keras_nlp")
```

Comment:
We are switching to an explicitly exported API, so we can move this to `@keras_nlp_export("keras_nlp.models.AlbertBackbone")`. The encoder/decoder blocks leave as is, we aren't exporting those for now.